Binary files /tmp/glade/.glade.SlackBuild.un~ and /source/l/glade/.glade.SlackBuild.un~ differ
diff -ruBN /tmp/glade/glade.SlackBuild.log /source/l/glade/glade.SlackBuild.log
--- /tmp/glade/glade.SlackBuild.log	2021-01-23 08:12:04.000000000 +0000
+++ /source/l/glade/glade.SlackBuild.log	2021-01-23 08:16:25.000000000 +0000
@@ -1096,37 +1096,37 @@
 [5/255] Generating glade-previewer.1 with a custom command
 [6/255] Generating glade-gtk-marshallers_h with a custom command
 INFO: Reading ../plugins/gtk+/glade-gtk-marshallers.list...
-[7/255] Generating glade-resources_h with a custom command
-[8/255] Generating glade-resources_c with a custom command
-[9/255] Generating glade.1 with a custom command
+[7/255] Generating glade.1 with a custom command
+[8/255] Generating glade-resources_h with a custom command
+[9/255] Generating glade-resources_c with a custom command
 [10/255] Generating glade-marshallers_c with a custom command
 INFO: Reading ../gladeui/glade-marshallers.list...
-[11/255] Generating glade-gtk-resources_c with a custom command
-[12/255] Generating gladeui-resources_c with a custom command
-[13/255] Generating gladeui-resources_h with a custom command
-[14/255] Generating glade-marshallers_h with a custom command
+[11/255] Generating glade-marshallers_h with a custom command
 INFO: Reading ../gladeui/glade-marshallers.list...
+[12/255] Generating gladeui-resources_c with a custom command
+[13/255] Generating glade-gtk-resources_c with a custom command
+[14/255] Generating gladeui-resources_h with a custom command
 [15/255] Generating org.gnome.Glade.appdata.xml_data_merge with a custom command
 [16/255] Generating org.gnome.Glade.desktop_data_merge with a custom command
 [17/255] Generating gladeui-enum-types.h with a custom command (wrapped by meson to capture output)
-[18/255] Compiling C object plugins/gladeui/libgladeglade.so.p/glade-glade-property-shell.c.o
+[18/255] Compiling C object plugins/python/libgladepython.so.p/glade-python.c.o
 [19/255] Compiling C object plugins/gladeui/libgladeglade.so.p/glade-glade-editor-skeleton.c.o
-[20/255] Compiling C object plugins/python/libgladepython.so.p/glade-python.c.o
+[20/255] Compiling C object plugins/gladeui/libgladeglade.so.p/glade-glade-property-shell.c.o
 [21/255] Compiling C object tests/add-child.p/add-child.c.o
-[22/255] Compiling C object tests/modules.p/modules.c.o
-[23/255] Compiling C object tests/create-widgets.p/create-widgets.c.o
-[24/255] Compiling C object tests/toplevel-order.p/meson-generated_.._toplevel-order-resources.c.o
-[25/255] Compiling C object tests/toplevel-order.p/toplevel-order.c.o
-[26/255] Compiling C object tests/refcount.p/refcount.c.o
-[27/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-model-data.c.o
-[28/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-model-button-editor.c.o
+[22/255] Compiling C object tests/create-widgets.p/create-widgets.c.o
+[23/255] Compiling C object tests/modules.p/modules.c.o
+[24/255] Compiling C object tests/refcount.p/refcount.c.o
+[25/255] Compiling C object tests/toplevel-order.p/meson-generated_.._toplevel-order-resources.c.o
+[26/255] Compiling C object tests/toplevel-order.p/toplevel-order.c.o
+[27/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-model-button-editor.c.o
+[28/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-model-data.c.o
 [29/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-notebook-editor.c.o
-[30/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-progress-bar-editor.c.o
-[31/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-popover-menu-editor.c.o
-[32/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-popover-editor.c.o
-[33/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-recent-action-editor.c.o
-[34/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-recent-chooser-dialog-editor.c.o
-[35/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-real-tree-view-editor.c.o
+[30/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-popover-menu-editor.c.o
+[31/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-popover-editor.c.o
+[32/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-progress-bar-editor.c.o
+[33/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-real-tree-view-editor.c.o
+[34/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-recent-action-editor.c.o
+[35/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-recent-chooser-dialog-editor.c.o
 [36/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-recent-chooser-editor.c.o
 [37/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-recent-chooser-menu-editor.c.o
 [38/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-recent-chooser-widget-editor.c.o
@@ -1135,36 +1135,36 @@
 [41/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-scrollable-editor.c.o
 [42/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-scrollbar-editor.c.o
 [43/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-scrolled-window-editor.c.o
-[44/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-stack-editor.c.o
-[45/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-spin-button-editor.c.o
+[44/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-spin-button-editor.c.o
+[45/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-stack-editor.c.o
 [46/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-stack-switcher-editor.c.o
 [47/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-store-editor.c.o
-[48/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-text-view-editor.c.o
-[49/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-string-list.c.o
-[50/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-tool-button-editor.c.o
+[48/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-string-list.c.o
+[49/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-text-view-editor.c.o
+[50/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-tool-palette-editor.c.o
 [51/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-tool-item-group-editor.c.o
-[52/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-tool-palette-editor.c.o
-[53/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-treeview-editor.c.o
-[54/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-viewport-editor.c.o
+[52/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-tool-button-editor.c.o
+[53/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-viewport-editor.c.o
+[54/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-treeview-editor.c.o
 [55/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-widget-editor.c.o
-[56/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-window-editor.c.o
-[57/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-recent-file-filter.c.o
+[56/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-recent-chooser-widget.c.o
+[57/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-window-editor.c.o
 [58/255] Compiling C object plugins/gjs/libgladegjs.so.p/glade-gjs.c.o
-[59/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-recent-chooser-widget.c.o
-[60/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-scrollbar.c.o
-[61/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-revealer.c.o
-[62/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-scale.c.o
+[59/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-recent-file-filter.c.o
+[60/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-revealer.c.o
+[61/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-scale.c.o
+[62/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-scrollbar.c.o
 [63/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-scrolled-window.c.o
-[64/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-size-group.c.o
-[65/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-searchbar.c.o
+[64/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-searchbar.c.o
+[65/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-size-group.c.o
 [66/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-stack.c.o
 [67/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-spin-button.c.o
-[68/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-stack-switcher.c.o
-[69/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-switch.c.o
+[68/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-switch.c.o
+[69/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-stack-switcher.c.o
 [70/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-table.c.o
 [71/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-text-buffer.c.o
-[72/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-text-tag-table.c.o
-[73/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-tool-button.c.o
+[72/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-tool-button.c.o
+[73/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-text-tag-table.c.o
 [74/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-text-view.c.o
 [75/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-tool-item.c.o
 [76/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-tool-item-group.c.o
@@ -1172,9 +1172,9 @@
 [78/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-toolbar.c.o
 [79/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-tree-view.c.o
 [80/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-viewport.c.o
-[81/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-widget.c.o
-[82/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-window.c.o
-[83/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-header-bar-editor.c.o
+[81/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-header-bar-editor.c.o
+[82/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-widget.c.o
+[83/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-window.c.o
 [84/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-icon-factory-editor.c.o
 [85/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-icon-sources.c.o
 [86/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-icon-view-editor.c.o
@@ -1183,80 +1183,80 @@
 [89/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-label-editor.c.o
 [90/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-layout-editor.c.o
 [91/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-level-bar-editor.c.o
-[92/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-message-dialog-editor.c.o
-[93/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-combo-box.c.o
-[94/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-misc-editor.c.o
+[92/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-misc-editor.c.o
+[93/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-message-dialog-editor.c.o
+[94/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-combo-box.c.o
 [95/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-combo-box-text.c.o
 [96/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-container.c.o
-[97/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-dialog.c.o
-[98/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-entry-buffer.c.o
-[99/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-entry.c.o
+[97/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-entry.c.o
+[98/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-dialog.c.o
+[99/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-entry-buffer.c.o
 [100/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-expander.c.o
-[101/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-file-chooser-widget.c.o
-[102/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-fixed-layout.c.o
-[103/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-flow-box.c.o
-[104/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-frame.c.o
-[105/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-font-chooser-widget.c.o
-[106/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-header-bar.c.o
-[107/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-grid.c.o
-[108/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-icon-factory.c.o
-[109/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-image.c.o
-[110/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-icon-view.c.o
+[101/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-flow-box.c.o
+[102/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-file-chooser-widget.c.o
+[103/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-fixed-layout.c.o
+[104/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-font-chooser-widget.c.o
+[105/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-frame.c.o
+[106/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-grid.c.o
+[107/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-header-bar.c.o
+[108/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-icon-view.c.o
+[109/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-icon-factory.c.o
+[110/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-image.c.o
 [111/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-image-menu-item.c.o
 [112/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-info-bar.c.o
-[113/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-level-bar.c.o
-[114/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-label.c.o
-[115/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-menu-bar.c.o
+[113/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-label.c.o
+[114/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-level-bar.c.o
+[115/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-list-box.c.o
 [116/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-list-store.c.o
-[117/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-list-box.c.o
-[118/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-menu-item.c.o
-[119/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-menu.c.o
+[117/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-menu-bar.c.o
+[118/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-menu.c.o
+[119/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-menu-item.c.o
 [120/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-menu-shell.c.o
-[121/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-model-button.c.o
-[122/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-menu-tool-button.c.o
-[123/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-message-dialog.c.o
+[121/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-menu-tool-button.c.o
+[122/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-message-dialog.c.o
+[123/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-model-button.c.o
 [124/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-notebook.c.o
 [125/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-overlay.c.o
 [126/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-paned.c.o
-[127/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-popover.c.o
-[128/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-popover-menu.c.o
-[129/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-progress-bar.c.o
+[127/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-progress-bar.c.o
+[128/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-popover.c.o
+[129/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-popover-menu.c.o
 [130/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-radio-button.c.o
 [131/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-radio-menu-item.c.o
 [132/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-recent-chooser-menu.c.o
-[133/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-action-bar.c.o
-[134/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-about-dialog.c.o
+[133/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-about-dialog.c.o
+[134/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-action-bar.c.o
 [135/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-action.c.o
-[136/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-adjustment.c.o
-[137/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-action-widgets.c.o
-[138/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-action-group.c.o
-[139/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-app-chooser-button.c.o
-[140/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-arrow.c.o
-[141/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-app-chooser-widget.c.o
+[136/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-action-group.c.o
+[137/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-app-chooser-button.c.o
+[138/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-action-widgets.c.o
+[139/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-adjustment.c.o
+[140/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-app-chooser-widget.c.o
+[141/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-arrow.c.o
 [142/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-assistant.c.o
-[143/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-bin.c.o
-[144/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-box.c.o
-[145/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-button.c.o
-[146/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-cell-renderer.c.o
-[147/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-cell-layout.c.o
-[148/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-cell-renderer-editor.c.o
-[149/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-button-editor.c.o
+[143/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-button.c.o
+[144/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-bin.c.o
+[145/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-box.c.o
+[146/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-cell-layout.c.o
+[147/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk-cell-renderer.c.o
+[148/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-button-editor.c.o
+[149/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-cell-renderer-editor.c.o
 [150/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-column-types.c.o
 [151/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-combo-box-editor.c.o
-[152/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-entry-editor.c.o
-[153/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-combo-box-text-editor.c.o
-[154/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-file-chooser-button-editor.c.o
-[155/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-eprop-enum-int.c.o
-[156/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-file-chooser-dialog-editor.c.o
+[152/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-combo-box-text-editor.c.o
+[153/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-entry-editor.c.o
+[154/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-eprop-enum-int.c.o
+[155/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-file-chooser-dialog-editor.c.o
+[156/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-file-chooser-button-editor.c.o
 [157/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-file-chooser-editor.c.o
 [158/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-file-chooser-widget-editor.c.o
-[159/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-fixed.c.o
-[160/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-font-button-editor.c.o
-[161/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-font-chooser-dialog-editor.c.o
+[159/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-font-chooser-dialog-editor.c.o
+[160/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-fixed.c.o
+[161/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-font-button-editor.c.o
 [162/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-font-chooser-editor.c.o
 [163/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-font-chooser-widget-editor.c.o
-[164/255] Compiling C object plugins/gtk+/libgladegtk.so.p/meson-generated_.._glade-gtk-marshallers.c.o
-[165/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-grid-editor.c.o
+[164/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-grid-editor.c.o
+[165/255] Compiling C object plugins/gtk+/libgladegtk.so.p/meson-generated_.._glade-gtk-marshallers.c.o
 [166/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-gtk.c.o
 [167/255] Compiling C object plugins/gtk+/libgladegtk.so.p/meson-generated_.._glade-gtk-resources.c.o
 [168/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-about-dialog-editor.c.o
@@ -1265,71 +1265,71 @@
 [171/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-activatable-editor.c.o
 [172/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-app-chooser-button-editor.c.o
 [173/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-app-chooser-widget-editor.c.o
-[174/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-action-bar-editor.c.o
-[175/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-arrow-editor.c.o
-[176/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-attributes.c.o
+[174/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-arrow-editor.c.o
+[175/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-attributes.c.o
+[176/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-action-bar-editor.c.o
 [177/255] Compiling C object plugins/gtk+/libgladegtk.so.p/glade-box-editor.c.o
-[178/255] Compiling C object gladeui/glade-previewer.p/glade-previewer.c.o
-[179/255] Compiling C object gladeui/glade-previewer.p/glade-previewer-main.c.o
+[178/255] Compiling C object gladeui/glade-previewer.p/glade-previewer-main.c.o
+[179/255] Compiling C object gladeui/glade-previewer.p/glade-previewer.c.o
 [180/255] Compiling C object src/glade.p/meson-generated_.._glade-resources.c.o
 [181/255] Compiling C object src/glade.p/glade-http.c.o
 [182/255] Compiling C object src/glade.p/glade-intro.c.o
-[183/255] Compiling C object src/glade.p/glade-registration.c.o
+[183/255] Compiling C object src/glade.p/glade-settings.c.o
 [184/255] Compiling C object src/glade.p/glade-preferences.c.o
-[185/255] Compiling C object src/glade.p/glade-settings.c.o
-[186/255] Compiling C object src/glade.p/glade-window.c.o
-[187/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-widget-action.c.o
+[185/255] Compiling C object src/glade.p/glade-registration.c.o
+[186/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-utils.c.o
+[187/255] Compiling C object src/glade.p/glade-window.c.o
 [188/255] Compiling C object src/glade.p/main.c.o
-[189/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-utils.c.o
+[189/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-widget-action.c.o
 [190/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-widget-adaptor.c.o
 [191/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-widget.c.o
 [192/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-xml-utils.c.o
 [193/255] Compiling C object gladeui/glade-previewer.p/glade-preview-template.c.o
 [194/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-project.c.o
 [195/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-project-properties.c.o
-[196/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-property-def.c.o
-[197/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-property.c.o
+[196/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-property.c.o
+[197/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-property-def.c.o
 [198/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-property-label.c.o
 [199/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-property-shell.c.o
-[200/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-signal.c.o
-[201/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-signal-editor.c.o
-[202/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-template.c.o
-[203/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-signal-def.c.o
-[204/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-signal-model.c.o
-[205/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-tsort.c.o
-[206/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-id-allocator.c.o
-[207/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-inspector.c.o
-[208/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-name-context.c.o
+[200/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-signal-def.c.o
+[201/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-signal.c.o
+[202/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-signal-editor.c.o
+[203/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-tsort.c.o
+[204/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-id-allocator.c.o
+[205/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-signal-model.c.o
+[206/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-template.c.o
+[207/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-name-context.c.o
+[208/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-inspector.c.o
 [209/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-named-icon-chooser-dialog.c.o
 [210/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-object-stub.c.o
 [211/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-palette.c.o
-[212/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-placeholder.c.o
-[213/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-preview.c.o
-[214/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-popup.c.o
+[212/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-popup.c.o
+[213/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-placeholder.c.o
+[214/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-preview.c.o
 [215/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-displayable-values.c.o
 [216/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-dnd.c.o
-[217/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-editable.c.o
+[217/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-design-view.c.o
 [218/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-drag.c.o
-[219/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-design-view.c.o
+[219/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-editable.c.o
 [220/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-editor.c.o
 [221/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-editor-property.c.o
-[222/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-editor-skeleton.c.o
-[223/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-editor-table.c.o
+[222/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-editor-table.c.o
+[223/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-editor-skeleton.c.o
 [224/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-cell-renderer-icon.c.o
-[225/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-clipboard.c.o
-[226/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-command.c.o
+[225/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-command.c.o
+[226/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-clipboard.c.o
 [227/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-cursor.c.o
 [228/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-debug.c.o
 [229/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-design-layout.c.o
-[230/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-adaptor-chooser-widget.c.o
-[231/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-app.c.o
-[232/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-base-editor.c.o
-[233/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-adaptor-chooser.c.o
-[234/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-accumulators.c.o
-[235/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-builtins.c.o
-[236/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-catalog.c.o
-[237/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/meson-generated_.._gladeui-resources.c.o
-[238/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/meson-generated_.._glade-marshallers.c.o
+[230/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-adaptor-chooser.c.o
+[231/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-adaptor-chooser-widget.c.o
+[232/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-app.c.o
+[233/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-base-editor.c.o
+[234/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-builtins.c.o
+[235/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-catalog.c.o
+[236/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/glade-accumulators.c.o
+[237/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/meson-generated_.._glade-marshallers.c.o
+[238/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/meson-generated_.._gladeui-resources.c.o
 [239/255] Generating gladeui-enum-types.c with a custom command (wrapped by meson to capture output)
 [240/255] Compiling C object gladeui/libgladeui-2.so.13.0.2.p/meson-generated_.._gladeui-enum-types.c.o
 [241/255] Linking target gladeui/libgladeui-2.so.13.0.2
@@ -1346,7 +1346,7 @@
 [252/255] Linking target gladeui/glade-previewer
 [253/255] Linking target src/glade
 [254/255] Generating Gladeui-2.0.gir with a custom command
-g-ir-scanner: link: gcc -o /tmp/glade-3.38.2/meson-build/tmp-introspectr_zdvghh/Gladeui-2.0 -O2 -fPIC /tmp/glade-3.38.2/meson-build/tmp-introspectr_zdvghh/Gladeui-2.0.o -L. -Wl,-rpath,. -Wl,--no-as-needed -L/tmp/glade-3.38.2/meson-build/gladeui -Wl,-rpath,/tmp/glade-3.38.2/meson-build/gladeui -lgladeui-2 -lglib-2.0 -lgmodule-2.0 -lgtk-3 -lgdk-3 -lz -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lxml2 -lm -lgirepository-1.0 -lgio-2.0 -lgobject-2.0 -Wl,--export-dynamic -lgmodule-2.0 -pthread -lglib-2.0
+g-ir-scanner: link: gcc -o /tmp/glade-3.38.2/meson-build/tmp-introspect31pn2qjl/Gladeui-2.0 -O2 -fPIC /tmp/glade-3.38.2/meson-build/tmp-introspect31pn2qjl/Gladeui-2.0.o -L. -Wl,-rpath,. -Wl,--no-as-needed -L/tmp/glade-3.38.2/meson-build/gladeui -Wl,-rpath,/tmp/glade-3.38.2/meson-build/gladeui -lgladeui-2 -lglib-2.0 -lgmodule-2.0 -lgtk-3 -lgdk-3 -lz -lpangocairo-1.0 -lpango-1.0 -lharfbuzz -latk-1.0 -lcairo-gobject -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lxml2 -lm -lgirepository-1.0 -lgio-2.0 -lgobject-2.0 -Wl,--export-dynamic -lgmodule-2.0 -pthread -lglib-2.0
 [255/255] Generating Gladeui-2.0.typelib with a custom command
 [0/1] Installing files.
 Installing data/org.gnome.Glade.desktop to /tmp/package-glade/usr/share/applications
@@ -1770,8 +1770,6 @@
 Installing /tmp/glade-3.38.2/meson-build/po/zh_CN.gmo to /tmp/package-glade/usr/share/locale/zh_CN/LC_MESSAGES/glade.mo
 Installing /tmp/glade-3.38.2/meson-build/po/zh_HK.gmo to /tmp/package-glade/usr/share/locale/zh_HK/LC_MESSAGES/glade.mo
 Installing /tmp/glade-3.38.2/meson-build/po/zh_TW.gmo to /tmp/package-glade/usr/share/locale/zh_TW/LC_MESSAGES/glade.mo
-fopen: No such file or directory
-Error: cannot open mo file /tmp/glade-3.38.2/meson-build/help/hi/hi.gmo
 Installing /tmp/glade-3.38.2/meson-build/help/da/index.docbook to /tmp/package-glade/usr/share/help/da/glade/index.docbook
 Installing /tmp/glade-3.38.2/meson-build/help/da/legal.xml to /tmp/package-glade/usr/share/help/da/glade/legal.xml
 Installing /tmp/glade-3.38.2/help/da/figures/main-window.png to /tmp/package-glade/usr/share/help/da/glade/figures/main-window.png
@@ -1808,25 +1806,48 @@
 Installing /tmp/glade-3.38.2/meson-build/help/gl/index.docbook to /tmp/package-glade/usr/share/help/gl/glade/index.docbook
 Installing /tmp/glade-3.38.2/meson-build/help/gl/legal.xml to /tmp/package-glade/usr/share/help/gl/glade/legal.xml
 Symlinking /tmp/package-glade/usr/share/help/gl/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
-Installing /tmp/glade-3.38.2/meson-build/help/hi/index.docbook to /tmp/package-glade/usr/share/help/hi/glade/index.docbook
-Traceback (most recent call last):
-  File "/usr/bin/meson", line 33, in <module>
-    sys.exit(load_entry_point('meson==0.56.2', 'console_scripts', 'meson')())
-  File "/usr/lib/python3.9/site-packages/mesonbuild/mesonmain.py", line 240, in main
-    return run(sys.argv[1:], launcher)
-  File "/usr/lib/python3.9/site-packages/mesonbuild/mesonmain.py", line 229, in run
-    return run_script_command(args[1], args[2:])
-  File "/usr/lib/python3.9/site-packages/mesonbuild/mesonmain.py", line 177, in run_script_command
-    return module.run(script_args)
-  File "/usr/lib/python3.9/site-packages/mesonbuild/scripts/yelphelper.py", line 131, in run
-    install_help(src_subdir, build_subdir, sources, media, langs, install_dir,
-  File "/usr/lib/python3.9/site-packages/mesonbuild/scripts/yelphelper.py", line 72, in install_help
-    shutil.copy2(infile, outfile)
-  File "/usr/lib/python3.9/shutil.py", line 435, in copy2
-    copyfile(src, dst, follow_symlinks=follow_symlinks)
-  File "/usr/lib/python3.9/shutil.py", line 264, in copyfile
-    with open(src, 'rb') as fsrc, open(dst, 'wb') as fdst:
-FileNotFoundError: [Errno 2] No such file or directory: '/tmp/glade-3.38.2/meson-build/help/hi/index.docbook'
+Installing /tmp/glade-3.38.2/meson-build/help/hu/index.docbook to /tmp/package-glade/usr/share/help/hu/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/hu/legal.xml to /tmp/package-glade/usr/share/help/hu/glade/legal.xml
+Symlinking /tmp/package-glade/usr/share/help/hu/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
+Installing /tmp/glade-3.38.2/meson-build/help/id/index.docbook to /tmp/package-glade/usr/share/help/id/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/id/legal.xml to /tmp/package-glade/usr/share/help/id/glade/legal.xml
+Symlinking /tmp/package-glade/usr/share/help/id/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
+Installing /tmp/glade-3.38.2/meson-build/help/it/index.docbook to /tmp/package-glade/usr/share/help/it/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/it/legal.xml to /tmp/package-glade/usr/share/help/it/glade/legal.xml
+Symlinking /tmp/package-glade/usr/share/help/it/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
+Installing /tmp/glade-3.38.2/meson-build/help/ja/index.docbook to /tmp/package-glade/usr/share/help/ja/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/ja/legal.xml to /tmp/package-glade/usr/share/help/ja/glade/legal.xml
+Symlinking /tmp/package-glade/usr/share/help/ja/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
+Installing /tmp/glade-3.38.2/meson-build/help/ko/index.docbook to /tmp/package-glade/usr/share/help/ko/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/ko/legal.xml to /tmp/package-glade/usr/share/help/ko/glade/legal.xml
+Symlinking /tmp/package-glade/usr/share/help/ko/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
+Installing /tmp/glade-3.38.2/meson-build/help/oc/index.docbook to /tmp/package-glade/usr/share/help/oc/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/oc/legal.xml to /tmp/package-glade/usr/share/help/oc/glade/legal.xml
+Symlinking /tmp/package-glade/usr/share/help/oc/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
+Installing /tmp/glade-3.38.2/meson-build/help/pl/index.docbook to /tmp/package-glade/usr/share/help/pl/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/pl/legal.xml to /tmp/package-glade/usr/share/help/pl/glade/legal.xml
+Installing /tmp/glade-3.38.2/help/pl/figures/main-window.png to /tmp/package-glade/usr/share/help/pl/glade/figures/main-window.png
+Installing /tmp/glade-3.38.2/meson-build/help/pt_BR/index.docbook to /tmp/package-glade/usr/share/help/pt_BR/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/pt_BR/legal.xml to /tmp/package-glade/usr/share/help/pt_BR/glade/legal.xml
+Installing /tmp/glade-3.38.2/help/pt_BR/figures/main-window.png to /tmp/package-glade/usr/share/help/pt_BR/glade/figures/main-window.png
+Installing /tmp/glade-3.38.2/meson-build/help/ru/index.docbook to /tmp/package-glade/usr/share/help/ru/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/ru/legal.xml to /tmp/package-glade/usr/share/help/ru/glade/legal.xml
+Installing /tmp/glade-3.38.2/help/ru/figures/main-window.png to /tmp/package-glade/usr/share/help/ru/glade/figures/main-window.png
+Installing /tmp/glade-3.38.2/meson-build/help/sl/index.docbook to /tmp/package-glade/usr/share/help/sl/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/sl/legal.xml to /tmp/package-glade/usr/share/help/sl/glade/legal.xml
+Symlinking /tmp/package-glade/usr/share/help/sl/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
+Installing /tmp/glade-3.38.2/meson-build/help/sv/index.docbook to /tmp/package-glade/usr/share/help/sv/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/sv/legal.xml to /tmp/package-glade/usr/share/help/sv/glade/legal.xml
+Installing /tmp/glade-3.38.2/help/sv/figures/main-window.png to /tmp/package-glade/usr/share/help/sv/glade/figures/main-window.png
+Installing /tmp/glade-3.38.2/meson-build/help/uk/index.docbook to /tmp/package-glade/usr/share/help/uk/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/uk/legal.xml to /tmp/package-glade/usr/share/help/uk/glade/legal.xml
+Symlinking /tmp/package-glade/usr/share/help/uk/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
+Installing /tmp/glade-3.38.2/meson-build/help/zh_CN/index.docbook to /tmp/package-glade/usr/share/help/zh_CN/glade/index.docbook
+Installing /tmp/glade-3.38.2/meson-build/help/zh_CN/legal.xml to /tmp/package-glade/usr/share/help/zh_CN/glade/legal.xml
+Symlinking /tmp/package-glade/usr/share/help/zh_CN/glade/figures/main-window.png to /usr/share/help/C/glade/figures/main-window.png.
+Installing /tmp/glade-3.38.2/help/C/index.docbook to /tmp/package-glade/usr/share/help/C/glade/index.docbook
+Installing /tmp/glade-3.38.2/help/C/legal.xml to /tmp/package-glade/usr/share/help/C/glade/legal.xml
+Installing /tmp/glade-3.38.2/help/C/figures/main-window.png to /tmp/package-glade/usr/share/help/C/glade/figures/main-window.png
 
 Installing /tmp/glade-3.38.2/plugins/gtk+/icons/22x22/widget-gtk-treemodelsort.png to /tmp/package-glade/usr/share/glade/pixmaps/hicolor/22x22/actions
 Installing /tmp/glade-3.38.2/plugins/gtk+/icons/22x22/widget-gtk-treeselection.png to /tmp/package-glade/usr/share/glade/pixmaps/hicolor/22x22/actions
@@ -1849,7 +1870,865 @@
 Installing /tmp/glade-3.38.2/plugins/glade-catalog.dtd to /tmp/package-glade/usr/share/glade/catalogs
 Running custom install script '/usr/bin/meson --internal gettext install --subdir=po --localedir=share/locale --pkgname=glade'
 Running custom install script '/usr/bin/meson --internal yelphelper install --subdir=help --id=glade --installdir=share/help --sources=index.docbook@@legal.xml --symlinks=true --media=figures/main-window.png'
-FAILED: install script '/usr/bin/meson --internal yelphelper install --subdir=help --id=glade --installdir=share/help --sources=index.docbook@@legal.xml --symlinks=true --media=figures/main-window.png' exit code 1, stopped
-FAILED: meson-install 
-/usr/bin/meson install --no-rebuild
-ninja: build stopped: subcommand failed.
+Running custom install script '/usr/bin/python3 /tmp/glade-3.38.2/meson_post_install.py share'
+cp: cannot stat 'INSTALL*': No such file or directory
+
+Slackware package maker, version 3.14159265.
+
+Searching for symbolic links:
+usr/lib/libgladeui-2.so	libgladeui-2.so.13
+usr/lib/libgladeui-2.so.13	libgladeui-2.so.13.0.2
+usr/share/help/bg/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/ca/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/el/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/en_GB/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/eu/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/gl/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/hu/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/id/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/it/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/ja/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/ko/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/oc/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/sl/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/uk/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+usr/share/help/zh_CN/glade/figures/main-window.png	/usr/share/help/C/glade/figures/main-window.png
+
+Making symbolic link creation script:
+( cd usr/lib ; rm -rf libgladeui-2.so )
+( cd usr/lib ; ln -sf libgladeui-2.so.13 libgladeui-2.so )
+( cd usr/lib ; rm -rf libgladeui-2.so.13 )
+( cd usr/lib ; ln -sf libgladeui-2.so.13.0.2 libgladeui-2.so.13 )
+( cd usr/share/help/bg/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/bg/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/ca/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/ca/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/el/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/el/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/en_GB/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/en_GB/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/eu/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/eu/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/gl/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/gl/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/hu/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/hu/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/id/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/id/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/it/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/it/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/ja/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/ja/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/ko/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/ko/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/oc/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/oc/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/sl/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/sl/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/uk/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/uk/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+( cd usr/share/help/zh_CN/glade/figures ; rm -rf main-window.png )
+( cd usr/share/help/zh_CN/glade/figures ; ln -sf /usr/share/help/C/glade/figures/main-window.png main-window.png )
+
+It is recommended that you make these lines your new installation script.
+
+Would you like to make this stuff the install script for this package
+and remove the symbolic links ([y]es, [n]o)? y
+
+
+Removing symbolic links:
+removed './usr/lib/libgladeui-2.so'
+removed './usr/lib/libgladeui-2.so.13'
+removed './usr/share/help/ja/glade/figures/main-window.png'
+removed './usr/share/help/uk/glade/figures/main-window.png'
+removed './usr/share/help/eu/glade/figures/main-window.png'
+removed './usr/share/help/ko/glade/figures/main-window.png'
+removed './usr/share/help/sl/glade/figures/main-window.png'
+removed './usr/share/help/oc/glade/figures/main-window.png'
+removed './usr/share/help/zh_CN/glade/figures/main-window.png'
+removed './usr/share/help/gl/glade/figures/main-window.png'
+removed './usr/share/help/el/glade/figures/main-window.png'
+removed './usr/share/help/en_GB/glade/figures/main-window.png'
+removed './usr/share/help/it/glade/figures/main-window.png'
+removed './usr/share/help/hu/glade/figures/main-window.png'
+removed './usr/share/help/bg/glade/figures/main-window.png'
+removed './usr/share/help/id/glade/figures/main-window.png'
+removed './usr/share/help/ca/glade/figures/main-window.png'
+
+Creating your new ./install/doinst.sh...
+
+This next step is optional - you can set the directories in your package
+to some sane permissions. If any of the directories in your package have
+special permissions, then DO NOT reset them here!
+
+Would you like to reset all directory permissions to 755 (drwxr-xr-x) and
+directory ownerships to root.root ([y]es, [n]o)? n
+
+Creating Slackware package:  /tmp/glade-3.38.2-x86_64-1.txz
+
+./
+install/
+install/doinst.sh
+install/slack-desc
+usr/
+usr/bin/
+usr/bin/glade
+usr/bin/glade-previewer
+usr/doc/
+usr/doc/glade-3.38.2/
+usr/doc/glade-3.38.2/AUTHORS
+usr/doc/glade-3.38.2/COPYING
+usr/doc/glade-3.38.2/COPYING.GPL
+usr/doc/glade-3.38.2/COPYING.LGPL
+usr/doc/glade-3.38.2/NEWS
+usr/doc/glade-3.38.2/README.md
+usr/doc/glade-3.38.2/TODO
+usr/include/
+usr/include/libgladeui-2.0/
+usr/include/libgladeui-2.0/gladeui/
+usr/include/libgladeui-2.0/gladeui/glade-adaptor-chooser.h
+usr/include/libgladeui-2.0/gladeui/glade-app.h
+usr/include/libgladeui-2.0/gladeui/glade-base-editor.h
+usr/include/libgladeui-2.0/gladeui/glade-builtins.h
+usr/include/libgladeui-2.0/gladeui/glade-catalog.h
+usr/include/libgladeui-2.0/gladeui/glade-cell-renderer-icon.h
+usr/include/libgladeui-2.0/gladeui/glade-clipboard.h
+usr/include/libgladeui-2.0/gladeui/glade-command.h
+usr/include/libgladeui-2.0/gladeui/glade-cursor.h
+usr/include/libgladeui-2.0/gladeui/glade-debug.h
+usr/include/libgladeui-2.0/gladeui/glade-design-view.h
+usr/include/libgladeui-2.0/gladeui/glade-displayable-values.h
+usr/include/libgladeui-2.0/gladeui/glade-editable.h
+usr/include/libgladeui-2.0/gladeui/glade-editor-property.h
+usr/include/libgladeui-2.0/gladeui/glade-editor-skeleton.h
+usr/include/libgladeui-2.0/gladeui/glade-editor-table.h
+usr/include/libgladeui-2.0/gladeui/glade-editor.h
+usr/include/libgladeui-2.0/gladeui/glade-inspector.h
+usr/include/libgladeui-2.0/gladeui/glade-name-context.h
+usr/include/libgladeui-2.0/gladeui/glade-palette.h
+usr/include/libgladeui-2.0/gladeui/glade-placeholder.h
+usr/include/libgladeui-2.0/gladeui/glade-project.h
+usr/include/libgladeui-2.0/gladeui/glade-property-def.h
+usr/include/libgladeui-2.0/gladeui/glade-property-label.h
+usr/include/libgladeui-2.0/gladeui/glade-property-shell.h
+usr/include/libgladeui-2.0/gladeui/glade-property.h
+usr/include/libgladeui-2.0/gladeui/glade-signal-def.h
+usr/include/libgladeui-2.0/gladeui/glade-signal-editor.h
+usr/include/libgladeui-2.0/gladeui/glade-signal-model.h
+usr/include/libgladeui-2.0/gladeui/glade-signal.h
+usr/include/libgladeui-2.0/gladeui/glade-utils.h
+usr/include/libgladeui-2.0/gladeui/glade-widget-action.h
+usr/include/libgladeui-2.0/gladeui/glade-widget-adaptor.h
+usr/include/libgladeui-2.0/gladeui/glade-widget.h
+usr/include/libgladeui-2.0/gladeui/glade-xml-utils.h
+usr/include/libgladeui-2.0/gladeui/glade.h
+usr/lib/
+usr/lib/girepository-1.0/
+usr/lib/girepository-1.0/Gladeui-2.0.typelib
+usr/lib/glade/
+usr/lib/glade/modules/
+usr/lib/glade/modules/libgladegjs.so
+usr/lib/glade/modules/libgladeglade.so
+usr/lib/glade/modules/libgladegtk.so
+usr/lib/glade/modules/libgladepython.so
+usr/lib/libgladeui-2.so.13.0.2
+usr/lib/pkgconfig/
+usr/lib/pkgconfig/gladeui-2.0.pc
+usr/man/
+usr/man/man1/
+usr/man/man1/glade-previewer.1
+usr/man/man1/glade.1
+usr/share/
+usr/share/applications/
+usr/share/applications/org.gnome.Glade.desktop
+usr/share/gettext/
+usr/share/gettext/its/
+usr/share/gettext/its/glade-catalog.its
+usr/share/gettext/its/glade-catalog.loc
+usr/share/gir-1.0/
+usr/share/gir-1.0/Gladeui-2.0.gir
+usr/share/glade/
+usr/share/glade/catalogs/
+usr/share/glade/catalogs/glade-catalog.dtd
+usr/share/glade/catalogs/gladeui.xml
+usr/share/glade/catalogs/gtk+.xml
+usr/share/glade/catalogs/gtkunixprint.xml
+usr/share/glade/pixmaps/
+usr/share/glade/pixmaps/deprecated-16x16.png
+usr/share/glade/pixmaps/deprecated-22x22.png
+usr/share/glade/pixmaps/devhelp.png
+usr/share/glade/pixmaps/fixed-bg.png
+usr/share/glade/pixmaps/hicolor/
+usr/share/glade/pixmaps/hicolor/16x16/
+usr/share/glade/pixmaps/hicolor/16x16/actions/
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-aboutdialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-accelgroup.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-accellabel.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-action.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-actionbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-actiongroup.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-adjustment.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-alignment.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-appchooserbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-appchooserdialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-appchooserwidget.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-arrow.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-aspectframe.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-assistant.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-box.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-button.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-buttonbox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-calendar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-cellareabox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-cellrendereraccel.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-cellrenderercombo.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-cellrendererpixbuf.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-cellrendererprogress.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-cellrendererspin.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-cellrendererspinner.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-cellrenderertext.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-cellrenderertoggle.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-checkbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-checkmenuitem.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-colorbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-colorselection.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-colorselectiondialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-combobox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-comboboxtext.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-default.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-dialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-drawingarea.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-entry.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-entrybuffer.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-entrycompletion.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-eventbox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-expander.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-filechooserbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-filechooserdialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-filechooserwidget.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-filefilter.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-fixed.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-flowbox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-flowboxchild.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-fontbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-fontselection.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-fontselectiondialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-frame.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-glarea.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-grid.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-handlebox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-hbox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-hbuttonbox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-headerbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-hpaned.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-hscale.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-hscrollbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-hseparator.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-iconfactory.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-iconview.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-image.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-imagemenuitem.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-infobar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-inputdialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-label.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-layout.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-levelbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-linkbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-listbox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-listboxrow.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-liststore.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-lockbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-menu.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-menubar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-menubutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-menuitem.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-menutoolbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-messagedialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-modelbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-notebook.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-offscreenwindow.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-overlay.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-pagesetupdialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-paned.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-placessidebar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-popover.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-popovermenu.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-printdialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-progressbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-radioaction.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-radiobutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-radiomenuitem.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-radiotoolbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-recentaction.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-recentchooser.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-recentchooserdialog.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-recentchoosermenu.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-recentfilter.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-recentmanager.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-revealer.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-scale.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-scalebutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-scrollbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-scrolledwindow.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-searchbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-searchentry.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-separator.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-separatormenuitem.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-separatortoolitem.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-sizegroup.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-spinbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-spinner.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-stack.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-stacksidebar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-stackswitcher.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-statusbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-statusicon.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-switch.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-table.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-tearoffmenuitem.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-textbuffer.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-texttag.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-texttagtable.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-textview.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-toggleaction.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-togglebutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-toggletoolbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-toolbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-toolbutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-toolitem.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-toolitemgroup.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-toolpalette.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-treemodelfilter.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-treemodelsort.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-treeselection.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-treestore.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-treeview.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-treeviewcolumn.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-vbox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-vbuttonbox.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-viewport.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-volumebutton.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-vpaned.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-vscale.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-vscrollbar.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-vseparator.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-window.png
+usr/share/glade/pixmaps/hicolor/16x16/actions/widget-gtk-windowgroup.png
+usr/share/glade/pixmaps/hicolor/22x22/
+usr/share/glade/pixmaps/hicolor/22x22/actions/
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-aboutdialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-accelgroup.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-accellabel.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-action.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-actionbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-actiongroup.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-adjustment.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-alignment.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-appchooserbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-appchooserdialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-appchooserwidget.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-arrow.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-aspectframe.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-assistant.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-box.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-button.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-buttonbox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-calendar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-cellareabox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-cellrendereraccel.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-cellrenderercombo.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-cellrendererpixbuf.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-cellrendererprogress.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-cellrendererspin.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-cellrendererspinner.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-cellrenderertext.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-cellrenderertoggle.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-checkbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-checkmenuitem.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-colorbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-colorselection.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-colorselectiondialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-combobox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-comboboxtext.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-default.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-dialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-drawingarea.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-entry.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-entrybuffer.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-entrycompletion.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-eventbox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-expander.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-filechooserbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-filechooserdialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-filechooserwidget.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-filefilter.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-fixed.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-flowbox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-flowboxchild.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-fontbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-fontselection.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-fontselectiondialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-frame.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-glarea.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-grid.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-handlebox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-hbox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-hbuttonbox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-headerbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-hpaned.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-hscale.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-hscrollbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-hseparator.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-iconfactory.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-iconview.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-image.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-imagemenuitem.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-infobar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-inputdialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-label.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-layout.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-levelbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-linkbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-listbox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-listboxrow.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-liststore.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-lockbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-menu.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-menubar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-menubutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-menuitem.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-menutoolbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-messagedialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-modelbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-notebook.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-offscreenwindow.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-overlay.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-pagesetupdialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-paned.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-placessidebar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-popover.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-popovermenu.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-printdialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-progressbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-radioaction.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-radiobutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-radiomenuitem.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-radiotoolbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-recentaction.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-recentchooser.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-recentchooserdialog.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-recentchoosermenu.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-recentfilter.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-recentmanager.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-revealer.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-scale.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-scalebutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-scrollbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-scrolledwindow.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-searchbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-searchentry.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-separator.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-separatormenuitem.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-separatortoolitem.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-sizegroup.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-spinbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-spinner.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-stack.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-stacksidebar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-stackswitcher.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-statusbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-statusicon.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-switch.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-table.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-tearoffmenuitem.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-textbuffer.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-texttag.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-texttagtable.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-textview.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-toggleaction.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-togglebutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-toggletoolbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-toolbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-toolbutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-toolitem.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-toolitemgroup.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-toolpalette.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-treemodelfilter.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-treemodelsort.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-treeselection.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-treestore.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-treeview.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-treeviewcolumn.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-vbox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-vbuttonbox.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-viewport.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-volumebutton.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-vpaned.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-vscale.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-vscrollbar.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-vseparator.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-window.png
+usr/share/glade/pixmaps/hicolor/22x22/actions/widget-gtk-windowgroup.png
+usr/share/glade/pixmaps/placeholder.png
+usr/share/glade/pixmaps/plus.png
+usr/share/help/
+usr/share/help/C/
+usr/share/help/C/glade/
+usr/share/help/C/glade/figures/
+usr/share/help/C/glade/figures/main-window.png
+usr/share/help/C/glade/index.docbook
+usr/share/help/C/glade/legal.xml
+usr/share/help/bg/
+usr/share/help/bg/glade/
+usr/share/help/bg/glade/figures/
+usr/share/help/bg/glade/index.docbook
+usr/share/help/bg/glade/legal.xml
+usr/share/help/ca/
+usr/share/help/ca/glade/
+usr/share/help/ca/glade/figures/
+usr/share/help/ca/glade/index.docbook
+usr/share/help/ca/glade/legal.xml
+usr/share/help/cs/
+usr/share/help/cs/glade/
+usr/share/help/cs/glade/figures/
+usr/share/help/cs/glade/figures/main-window.png
+usr/share/help/cs/glade/index.docbook
+usr/share/help/cs/glade/legal.xml
+usr/share/help/da/
+usr/share/help/da/glade/
+usr/share/help/da/glade/figures/
+usr/share/help/da/glade/figures/main-window.png
+usr/share/help/da/glade/index.docbook
+usr/share/help/da/glade/legal.xml
+usr/share/help/de/
+usr/share/help/de/glade/
+usr/share/help/de/glade/figures/
+usr/share/help/de/glade/figures/main-window.png
+usr/share/help/de/glade/index.docbook
+usr/share/help/de/glade/legal.xml
+usr/share/help/el/
+usr/share/help/el/glade/
+usr/share/help/el/glade/figures/
+usr/share/help/el/glade/index.docbook
+usr/share/help/el/glade/legal.xml
+usr/share/help/en_GB/
+usr/share/help/en_GB/glade/
+usr/share/help/en_GB/glade/figures/
+usr/share/help/en_GB/glade/index.docbook
+usr/share/help/en_GB/glade/legal.xml
+usr/share/help/es/
+usr/share/help/es/glade/
+usr/share/help/es/glade/figures/
+usr/share/help/es/glade/figures/main-window.png
+usr/share/help/es/glade/index.docbook
+usr/share/help/es/glade/legal.xml
+usr/share/help/eu/
+usr/share/help/eu/glade/
+usr/share/help/eu/glade/figures/
+usr/share/help/eu/glade/index.docbook
+usr/share/help/eu/glade/legal.xml
+usr/share/help/fr/
+usr/share/help/fr/glade/
+usr/share/help/fr/glade/figures/
+usr/share/help/fr/glade/figures/main-window.png
+usr/share/help/fr/glade/index.docbook
+usr/share/help/fr/glade/legal.xml
+usr/share/help/gl/
+usr/share/help/gl/glade/
+usr/share/help/gl/glade/figures/
+usr/share/help/gl/glade/index.docbook
+usr/share/help/gl/glade/legal.xml
+usr/share/help/hu/
+usr/share/help/hu/glade/
+usr/share/help/hu/glade/figures/
+usr/share/help/hu/glade/index.docbook
+usr/share/help/hu/glade/legal.xml
+usr/share/help/id/
+usr/share/help/id/glade/
+usr/share/help/id/glade/figures/
+usr/share/help/id/glade/index.docbook
+usr/share/help/id/glade/legal.xml
+usr/share/help/it/
+usr/share/help/it/glade/
+usr/share/help/it/glade/figures/
+usr/share/help/it/glade/index.docbook
+usr/share/help/it/glade/legal.xml
+usr/share/help/ja/
+usr/share/help/ja/glade/
+usr/share/help/ja/glade/figures/
+usr/share/help/ja/glade/index.docbook
+usr/share/help/ja/glade/legal.xml
+usr/share/help/ko/
+usr/share/help/ko/glade/
+usr/share/help/ko/glade/figures/
+usr/share/help/ko/glade/index.docbook
+usr/share/help/ko/glade/legal.xml
+usr/share/help/oc/
+usr/share/help/oc/glade/
+usr/share/help/oc/glade/figures/
+usr/share/help/oc/glade/index.docbook
+usr/share/help/oc/glade/legal.xml
+usr/share/help/pl/
+usr/share/help/pl/glade/
+usr/share/help/pl/glade/figures/
+usr/share/help/pl/glade/figures/main-window.png
+usr/share/help/pl/glade/index.docbook
+usr/share/help/pl/glade/legal.xml
+usr/share/help/pt_BR/
+usr/share/help/pt_BR/glade/
+usr/share/help/pt_BR/glade/figures/
+usr/share/help/pt_BR/glade/figures/main-window.png
+usr/share/help/pt_BR/glade/index.docbook
+usr/share/help/pt_BR/glade/legal.xml
+usr/share/help/ru/
+usr/share/help/ru/glade/
+usr/share/help/ru/glade/figures/
+usr/share/help/ru/glade/figures/main-window.png
+usr/share/help/ru/glade/index.docbook
+usr/share/help/ru/glade/legal.xml
+usr/share/help/sl/
+usr/share/help/sl/glade/
+usr/share/help/sl/glade/figures/
+usr/share/help/sl/glade/index.docbook
+usr/share/help/sl/glade/legal.xml
+usr/share/help/sv/
+usr/share/help/sv/glade/
+usr/share/help/sv/glade/figures/
+usr/share/help/sv/glade/figures/main-window.png
+usr/share/help/sv/glade/index.docbook
+usr/share/help/sv/glade/legal.xml
+usr/share/help/uk/
+usr/share/help/uk/glade/
+usr/share/help/uk/glade/figures/
+usr/share/help/uk/glade/index.docbook
+usr/share/help/uk/glade/legal.xml
+usr/share/help/zh_CN/
+usr/share/help/zh_CN/glade/
+usr/share/help/zh_CN/glade/figures/
+usr/share/help/zh_CN/glade/index.docbook
+usr/share/help/zh_CN/glade/legal.xml
+usr/share/icons/
+usr/share/icons/hicolor/
+usr/share/icons/hicolor/scalable/
+usr/share/icons/hicolor/scalable/apps/
+usr/share/icons/hicolor/scalable/apps/org.gnome.Glade.svg
+usr/share/icons/hicolor/symbolic/
+usr/share/icons/hicolor/symbolic/apps/
+usr/share/icons/hicolor/symbolic/apps/glade-brand-symbolic.svg
+usr/share/icons/hicolor/symbolic/apps/org.gnome.Glade-symbolic.svg
+usr/share/locale/
+usr/share/locale/ar/
+usr/share/locale/ar/LC_MESSAGES/
+usr/share/locale/ar/LC_MESSAGES/glade.mo
+usr/share/locale/ast/
+usr/share/locale/ast/LC_MESSAGES/
+usr/share/locale/ast/LC_MESSAGES/glade.mo
+usr/share/locale/az/
+usr/share/locale/az/LC_MESSAGES/
+usr/share/locale/az/LC_MESSAGES/glade.mo
+usr/share/locale/bg/
+usr/share/locale/bg/LC_MESSAGES/
+usr/share/locale/bg/LC_MESSAGES/glade.mo
+usr/share/locale/bn/
+usr/share/locale/bn/LC_MESSAGES/
+usr/share/locale/bn/LC_MESSAGES/glade.mo
+usr/share/locale/bn_IN/
+usr/share/locale/bn_IN/LC_MESSAGES/
+usr/share/locale/bn_IN/LC_MESSAGES/glade.mo
+usr/share/locale/bs/
+usr/share/locale/bs/LC_MESSAGES/
+usr/share/locale/bs/LC_MESSAGES/glade.mo
+usr/share/locale/ca/
+usr/share/locale/ca/LC_MESSAGES/
+usr/share/locale/ca/LC_MESSAGES/glade.mo
+usr/share/locale/ca@valencia/
+usr/share/locale/ca@valencia/LC_MESSAGES/
+usr/share/locale/ca@valencia/LC_MESSAGES/glade.mo
+usr/share/locale/cs/
+usr/share/locale/cs/LC_MESSAGES/
+usr/share/locale/cs/LC_MESSAGES/glade.mo
+usr/share/locale/da/
+usr/share/locale/da/LC_MESSAGES/
+usr/share/locale/da/LC_MESSAGES/glade.mo
+usr/share/locale/de/
+usr/share/locale/de/LC_MESSAGES/
+usr/share/locale/de/LC_MESSAGES/glade.mo
+usr/share/locale/el/
+usr/share/locale/el/LC_MESSAGES/
+usr/share/locale/el/LC_MESSAGES/glade.mo
+usr/share/locale/en@shaw/
+usr/share/locale/en@shaw/LC_MESSAGES/
+usr/share/locale/en@shaw/LC_MESSAGES/glade.mo
+usr/share/locale/en_CA/
+usr/share/locale/en_CA/LC_MESSAGES/
+usr/share/locale/en_CA/LC_MESSAGES/glade.mo
+usr/share/locale/en_GB/
+usr/share/locale/en_GB/LC_MESSAGES/
+usr/share/locale/en_GB/LC_MESSAGES/glade.mo
+usr/share/locale/eo/
+usr/share/locale/eo/LC_MESSAGES/
+usr/share/locale/eo/LC_MESSAGES/glade.mo
+usr/share/locale/es/
+usr/share/locale/es/LC_MESSAGES/
+usr/share/locale/es/LC_MESSAGES/glade.mo
+usr/share/locale/et/
+usr/share/locale/et/LC_MESSAGES/
+usr/share/locale/et/LC_MESSAGES/glade.mo
+usr/share/locale/eu/
+usr/share/locale/eu/LC_MESSAGES/
+usr/share/locale/eu/LC_MESSAGES/glade.mo
+usr/share/locale/fi/
+usr/share/locale/fi/LC_MESSAGES/
+usr/share/locale/fi/LC_MESSAGES/glade.mo
+usr/share/locale/fr/
+usr/share/locale/fr/LC_MESSAGES/
+usr/share/locale/fr/LC_MESSAGES/glade.mo
+usr/share/locale/fur/
+usr/share/locale/fur/LC_MESSAGES/
+usr/share/locale/fur/LC_MESSAGES/glade.mo
+usr/share/locale/gl/
+usr/share/locale/gl/LC_MESSAGES/
+usr/share/locale/gl/LC_MESSAGES/glade.mo
+usr/share/locale/gu/
+usr/share/locale/gu/LC_MESSAGES/
+usr/share/locale/gu/LC_MESSAGES/glade.mo
+usr/share/locale/he/
+usr/share/locale/he/LC_MESSAGES/
+usr/share/locale/he/LC_MESSAGES/glade.mo
+usr/share/locale/hu/
+usr/share/locale/hu/LC_MESSAGES/
+usr/share/locale/hu/LC_MESSAGES/glade.mo
+usr/share/locale/hy/
+usr/share/locale/hy/LC_MESSAGES/
+usr/share/locale/hy/LC_MESSAGES/glade.mo
+usr/share/locale/id/
+usr/share/locale/id/LC_MESSAGES/
+usr/share/locale/id/LC_MESSAGES/glade.mo
+usr/share/locale/it/
+usr/share/locale/it/LC_MESSAGES/
+usr/share/locale/it/LC_MESSAGES/glade.mo
+usr/share/locale/ja/
+usr/share/locale/ja/LC_MESSAGES/
+usr/share/locale/ja/LC_MESSAGES/glade.mo
+usr/share/locale/kk/
+usr/share/locale/kk/LC_MESSAGES/
+usr/share/locale/kk/LC_MESSAGES/glade.mo
+usr/share/locale/km/
+usr/share/locale/km/LC_MESSAGES/
+usr/share/locale/km/LC_MESSAGES/glade.mo
+usr/share/locale/ko/
+usr/share/locale/ko/LC_MESSAGES/
+usr/share/locale/ko/LC_MESSAGES/glade.mo
+usr/share/locale/lt/
+usr/share/locale/lt/LC_MESSAGES/
+usr/share/locale/lt/LC_MESSAGES/glade.mo
+usr/share/locale/lv/
+usr/share/locale/lv/LC_MESSAGES/
+usr/share/locale/lv/LC_MESSAGES/glade.mo
+usr/share/locale/mai/
+usr/share/locale/mai/LC_MESSAGES/
+usr/share/locale/mai/LC_MESSAGES/glade.mo
+usr/share/locale/mk/
+usr/share/locale/mk/LC_MESSAGES/
+usr/share/locale/mk/LC_MESSAGES/glade.mo
+usr/share/locale/ml/
+usr/share/locale/ml/LC_MESSAGES/
+usr/share/locale/ml/LC_MESSAGES/glade.mo
+usr/share/locale/mr/
+usr/share/locale/mr/LC_MESSAGES/
+usr/share/locale/mr/LC_MESSAGES/glade.mo
+usr/share/locale/ms/
+usr/share/locale/ms/LC_MESSAGES/
+usr/share/locale/ms/LC_MESSAGES/glade.mo
+usr/share/locale/nb/
+usr/share/locale/nb/LC_MESSAGES/
+usr/share/locale/nb/LC_MESSAGES/glade.mo
+usr/share/locale/ne/
+usr/share/locale/ne/LC_MESSAGES/
+usr/share/locale/ne/LC_MESSAGES/glade.mo
+usr/share/locale/nl/
+usr/share/locale/nl/LC_MESSAGES/
+usr/share/locale/nl/LC_MESSAGES/glade.mo
+usr/share/locale/nn/
+usr/share/locale/nn/LC_MESSAGES/
+usr/share/locale/nn/LC_MESSAGES/glade.mo
+usr/share/locale/oc/
+usr/share/locale/oc/LC_MESSAGES/
+usr/share/locale/oc/LC_MESSAGES/glade.mo
+usr/share/locale/or/
+usr/share/locale/or/LC_MESSAGES/
+usr/share/locale/or/LC_MESSAGES/glade.mo
+usr/share/locale/pa/
+usr/share/locale/pa/LC_MESSAGES/
+usr/share/locale/pa/LC_MESSAGES/glade.mo
+usr/share/locale/pl/
+usr/share/locale/pl/LC_MESSAGES/
+usr/share/locale/pl/LC_MESSAGES/glade.mo
+usr/share/locale/pt/
+usr/share/locale/pt/LC_MESSAGES/
+usr/share/locale/pt/LC_MESSAGES/glade.mo
+usr/share/locale/pt_BR/
+usr/share/locale/pt_BR/LC_MESSAGES/
+usr/share/locale/pt_BR/LC_MESSAGES/glade.mo
+usr/share/locale/ro/
+usr/share/locale/ro/LC_MESSAGES/
+usr/share/locale/ro/LC_MESSAGES/glade.mo
+usr/share/locale/ru/
+usr/share/locale/ru/LC_MESSAGES/
+usr/share/locale/ru/LC_MESSAGES/glade.mo
+usr/share/locale/si/
+usr/share/locale/si/LC_MESSAGES/
+usr/share/locale/si/LC_MESSAGES/glade.mo
+usr/share/locale/sk/
+usr/share/locale/sk/LC_MESSAGES/
+usr/share/locale/sk/LC_MESSAGES/glade.mo
+usr/share/locale/sl/
+usr/share/locale/sl/LC_MESSAGES/
+usr/share/locale/sl/LC_MESSAGES/glade.mo
+usr/share/locale/sq/
+usr/share/locale/sq/LC_MESSAGES/
+usr/share/locale/sq/LC_MESSAGES/glade.mo
+usr/share/locale/sr/
+usr/share/locale/sr/LC_MESSAGES/
+usr/share/locale/sr/LC_MESSAGES/glade.mo
+usr/share/locale/sr@latin/
+usr/share/locale/sr@latin/LC_MESSAGES/
+usr/share/locale/sr@latin/LC_MESSAGES/glade.mo
+usr/share/locale/sv/
+usr/share/locale/sv/LC_MESSAGES/
+usr/share/locale/sv/LC_MESSAGES/glade.mo
+usr/share/locale/ta/
+usr/share/locale/ta/LC_MESSAGES/
+usr/share/locale/ta/LC_MESSAGES/glade.mo
+usr/share/locale/te/
+usr/share/locale/te/LC_MESSAGES/
+usr/share/locale/te/LC_MESSAGES/glade.mo
+usr/share/locale/th/
+usr/share/locale/th/LC_MESSAGES/
+usr/share/locale/th/LC_MESSAGES/glade.mo
+usr/share/locale/tr/
+usr/share/locale/tr/LC_MESSAGES/
+usr/share/locale/tr/LC_MESSAGES/glade.mo
+usr/share/locale/ug/
+usr/share/locale/ug/LC_MESSAGES/
+usr/share/locale/ug/LC_MESSAGES/glade.mo
+usr/share/locale/uk/
+usr/share/locale/uk/LC_MESSAGES/
+usr/share/locale/uk/LC_MESSAGES/glade.mo
+usr/share/locale/vi/
+usr/share/locale/vi/LC_MESSAGES/
+usr/share/locale/vi/LC_MESSAGES/glade.mo
+usr/share/locale/zh_CN/
+usr/share/locale/zh_CN/LC_MESSAGES/
+usr/share/locale/zh_CN/LC_MESSAGES/glade.mo
+usr/share/locale/zh_HK/
+usr/share/locale/zh_HK/LC_MESSAGES/
+usr/share/locale/zh_HK/LC_MESSAGES/glade.mo
+usr/share/locale/zh_TW/
+usr/share/locale/zh_TW/LC_MESSAGES/
+usr/share/locale/zh_TW/LC_MESSAGES/glade.mo
+usr/share/metainfo/
+usr/share/metainfo/org.gnome.Glade.appdata.xml
+
+Slackware package /tmp/glade-3.38.2-x86_64-1.txz created.
+
